liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/types/Features.java

```diff
@@ -59,6 +60,9 @@ public class Features {
      */
     @Parameter(property = "from")
     private String from = null;
+    
+    @Parameter(property = "verify")
+    private String verify = "enforce";
```

Comment:
The default value is usually specified along with the property name such as:

```
@Parameter(property = "verify", defaultValue = "enforce")
```

Reply:
I still had to assign a value `enforce`. If I don't specify, the value was `null`.